tests/testsuite/update.rs

```diff
@@ -2255,14 +2218,10 @@ fn update_precise_breaking_spec_version() {
     // No spec
     p.cargo("update -Zunstable-options incompatible --precise 0.3.1")
```

Comment:
> `update_precise_breaking_spec_version`

I'm confused as to what this test is trying to capture compared to other tests

Reply:
Oh, its not that there is no spec but that we aren't specifying a version in the spec, only the name